gapic/src/main/com/google/gapid/widgets/Theme.java

```diff
@@ -207,6 +207,8 @@
   @TextStyle(foreground = 0xa9a9a9) public Styler structureStyler();
   @TextStyle(foreground = 0x0000ee) public Styler identifierStyler();
   @TextStyle(bold = true) public Styler labelStyler();
+  @TextStyle(bold = true, strikeout = true) public Styler disabledlabelStyler();
+  @TextStyle(bold = true, foreground = 0xa9a9a9) public Styler semiDisabledlabelStyler();
```

Comment:
Can we do a dotted line strikeout instead? I don't like that we use two different ways to communicate a similar thing.

Reply:
Sure.

Reply:
Do we have a dotted line strikeout in our library? I couldn't find one.

Reply:
Do we have a dotted line strikeout in our library? I couldn't find one.